netmiko/base_connection.py

```diff
@@ -123,6 +129,15 @@ def __init__(self, ip='', host='', username='', password='', secret='', port=Non
         self.timeout = timeout
         self.session_timeout = session_timeout
         self.keepalive = keepalive
+        if serial_settings is None:
+            serial_settings = {}
+        self.serial_settings = serial_settings
+        self.serial_defaults = {
+                'baudrate': 9600,
+                'bytesize': serial.EIGHTBITS,
+                'parity': serial.PARITY_NONE,
+                'stopbits': serial.STOPBITS_ONE
+            }
```

Comment:
Can we just do the following?

```
if serial_settings is not None:
    self.serial_settings = {
              'baudrate': 9600,
              'bytesize': serial.EIGHTBITS,
              'parity': serial.PARITY_NONE,
              'stopbits': serial.STOPBITS_ONE
            }
    self.serial_settings.update(serial_settings)
```

In other words, set the defaults and then update any defaults that were over-ridden in `__init__`.